### PR TITLE
feat(icons): added `circle-fading-arrow-up` icon

### PR DESCRIPTION
**Pull request by mosch**
## What is the purpose of this pull request?
- [x] New Icon
- [ ] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [ ] Other:

### Description
Adding circle-fading-up icon, a combination of two icons

### Icon use case 
- Premium Upgrade
- Fill out to complete something
- Data Refresh 
- Upgrade / Update App
- Indicating that there is a newer version of something available

### Concept 
- [x] I have provided valid use cases for each icon.
- [x] I have not added any a brand or logo icon.
- [x] I have not used any hate symbols.
- [x] I have not included any religious or political imagery.

### Author, credits & license
- [x] The icons are solely my own creation.
- [ ] The icons were originally created in #<issueNumber> by @<githubUser>
- [x] I've based them on the following Lucide icons: circle-fading-plus, circle-arrow-up
- [ ] I've based them on the following design: 

### Naming 

- [x] I've read and followed the [naming conventions](https://lucide.dev/guide/design/icon-design-guide#naming-conventions)
- [x] I've named icons by what they are rather than their use case.
- [x] I've provided meta JSON files in `icons/[iconName].json`.

### Design 

- [x] I've read and followed the [icon design guidelines](https://lucide.dev/guide/design/icon-design-guide)
- [x] I've made sure that the icons look sharp on low DPI displays.
- [x] I've made sure that the icons look consistent with the icon set in size, optical volume and density.
- [x] I've made sure that the icons are visually centered.
- [x] I've correctly optimized all icons to three points of precision.

## Before Submitting 

- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

**Review by karsa-mistmere — APPROVED** [review]

**Review by jguddas — APPROVED** [review]

File: icons/circle-fading-arrow-up.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "up",
+    "upgrade",
+    "improve",
+    "circle",
+    "button"
+  ],
+  "categories": [
+    "arrows",
+    "communication",
+    "shapes"
```
Comment (ericfennis) [inline icons/circle-fading-arrow-up.json:21]: These categories are not the correct ones. 
```suggestion
```

**Review by ericfennis — CHANGES_REQUESTED** [review]
@mosch Excellent icon, well done!
One small change for meta file.

File: icons/circle-fading-arrow-up.json (hunk 1 above)
Comment (karsa-mistmere) [reply icons/circle-fading-arrow-up.json:21]: `circle-fading-plus` also has `shapes` as a category, so I think that should be fine. `communication` I'm not 100% sold on either.

**Comment by mosch** [comment]
added development as category

**Comment by mosch** [comment]
@ericfennis let me know if we're good to go

**Review by ericfennis — APPROVED** [review]

**Pull request merged**